packages/peregrine/lib/talons/Gallery/useAddToCartButton.js

```diff
@@ -21,10 +21,10 @@ import operations from './addToCart.gql';
  *
  */
 const UNSUPPORTED_PRODUCT_TYPES = [
```

Comment:
Would be good to swap the logic around here as product types can be custom per store.

Reply:
I agree @fooman, but I think it would be out of the scope of this task.